src/app/_types/cmsConfig.ts

```diff
@@ -1,4 +1,4 @@
-export type CMSCollectionName = 'blog' | 'ecosystem' | 'events'
+export type CMSCollectionName = 'blog_posts' | 'ecosystem' | 'events'
```

Comment:
Shouldn't `events` be `event-entries` now if we follow the same logic?

Also, I think `events` is a self-explanatory name.

Reply:
Yes, it should be `event_entries`. The problem with just using `events` is that we also have the Events page, which is also called `events` in the CMS config.

Reply:
Actually, on my side it's already `event_entries` - Probably a CR that didn't get there in time.